search/search-dev-tools/src/components/query/style.scss

```diff
@@ -116,6 +115,7 @@ pre[class*='language-'] {
 	grid-area: query-src;
 	background-color: #ffffff;
 	position: relative;
+	max-width: calc( 50vw - 32px );
```

Comment:
We could have `50vw - 32px` as a constant. Also, it could be great if we put `32px` in its `vw` equivalent.

Reply:
Good point on using a variable. 

> Also, it could be great if we put 32px in its vw equivalent.
`vw` is a relative unit (1% of the viewport width) so it's not directly comparable, that's why we're using `calc` - to explicitly set the dimensions to 50% of a viewport minus 32px (which is a gutter/spacing constant that's used across the design).